src/plugins/intel_cpu/tools/commit_slider/utils/modes.py

```diff
@@ -284,6 +284,160 @@ def getCommitInfo(self, commit):
                 ci=super().getCommitInfo(commit),
                 d=commit.perfRel)
 
+
+class LLMBenchMode(Mode):
```

Comment:
Minor: please consider moving each mode to a separate file (since now `modes.py` is already quite cumbersome)